engine/physics/src/physics/physics_2d.h

```diff
@@ -105,6 +105,13 @@ namespace dmPhysics
         pp *= scale;
         return b2Mul(t, pp);
     }
+
+    inline b2Vec2 FromTransformScaleB2(const b2Transform& t, float inv_scale, const b2Vec2& p)
+    {
+        b2Vec2 pp = p;
+        pp *= inv_scale;
+        return b2MulT(t, pp);
+    }
```

Comment:
helper function for inverse transform

Opposite to `TransformScaleB2()` as we have `ToB2()`/`FromB2()`